src/tty_wrap.cc

```diff
@@ -61,9 +62,11 @@ void TTYWrap::Initialize(Local<Object> target,
   env->SetMethodNoSideEffect(target, "isTTY", IsTTY);
   env->SetMethodNoSideEffect(target, "guessHandleType", GuessHandleType);
 
-  target->Set(env->context(),
-              ttyString,
-              t->GetFunction(env->context()).ToLocalChecked()).FromJust();
+  MaybeLocal<Function> mfunc = t->GetFunction(env->context());
+  Local<Value> func = v8::Null(env->isolate());
```

Comment:
You don't need to assign, just `Local<Value> func` is enough. Idiomatic would be:
```c++
Local<Value> func_v;
if (t->GetFunction(env->context()).ToLocal(&func_v))
  target->Set(...).FromJust();
```
The `.FromJust()` is kind of questionable too because `Set()` can fail when near the stack limit. You could just ignore the return value:
```c++
// Can't do anything sensible on failure so we ignore the result.
USE(target->Set(...));
```

Reply:
I think we usually go with something like

```cpp
Local<Value> func;
if (t->GetFunction(env->context()).ToLocal(&func)) &&
  !target->Set(env->context(), ttyString, func).IsNothing()) {
  env->set_tty_constructor_template(t);
}
```

Reply:
Done.